src/test/java/co/rsk/federate/signing/utils/TestUtils.java

```diff
@@ -32,7 +33,11 @@
 import org.ethereum.core.Block;
 import org.ethereum.core.BlockHeader;
 
-public class TestUtils {
+public final class TestUtils {
+    private static final long CREATION_BLOCK_NUMBER = 0;
```

Comment:
Why make this a class variable? It only applies to a couple of methods

Reply:
If we put it on each method, it might be repeating code, on the other hand, if we put the value hardcoded, it is not a good practice since it's better to avoid magic numbers in the code.

Reply:
I'd prefer to have it repeated. This class has multiple utils and most of them won't make any use of this value

Reply:
Done!